ignition-server/controllers/local_ignitionprovider.go

```diff
@@ -759,7 +759,7 @@ func copyFile(src, dst string) error {
 }
 
 func invokeFeatureGateRenderScript(binary, workDir, outputDir string, payloadVersion semver.Version, featureGateYAML string) string {
-	var script = `#!/bin/bash
+	script := `#!/bin/bash
```

Comment:
why this change?

Reply:
This was from an auto formatting change that I missed after removing my other changes. I can revert this.